hdinsight-with-edge-node/readme.md

```diff
@@ -0,0 +1,8 @@
+# Create HDInsight Cluster with Edge Node
+
+Create HDInsight Cluster with Edge Node -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-with-edge-node%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Template creates an HDInsight cluster in a virtual network with another vm as an edge node that is bootstrapped with the clusters configurations.
```

Comment:
This readme will soon be rendered on Azure.com. It would be a good idea to explain the template in bit more detail and supply links for folks to learn more.